resources/saas-boost.yaml

```diff
@@ -83,6 +83,9 @@ Conditions:
   ShouldCreateMacroResources: !Equals [!Ref CreateMacroResources, 'true']
   UseCognito: !Equals [!Ref SystemIdentityProvider, 'COGNITO']
   UseKeycloak: !Equals [!Ref SystemIdentityProvider, 'KEYCLOAK']
+  IsCNRegion: !Or
+    - !Equals [ !Sub '${AWS::Region}', 'cn-north-1' ]
+    - !Equals [ !Sub '${AWS::Region}', 'cn-northwest-1' ]
```

Comment:
It is more forward-compatible to instead check that `${AWS::Partition}` is equal to `aws-cn`. Adding more regions is more likely than adding new partitions, and we don't want this to break if new regions are added.

Reply:
changed it to `IsCNRegion: !Equals [!Ref AWS::Partition, 'aws-cn']`